### PR TITLE
Fix some docs, remove outdated `accpetance-test` doc mentions and file

### PR DESCRIPTION
**Pull request by evantahler**
I don't expect the klaus-api tests to pass... this will be manually merged

### CODE REVIEW

File: airbyte-ci/connectors/pipelines/README.md [hunk 1]
```diff
@@ -81,73 +91,77 @@ However, this will not automatically install the dependencies for the local depe
...
-| `--ci-context`                                 | `manual`                        |                               | The current CI context: `manual` for manual run, `pull_request`, `nightly_builds`, `master`      |
-| `--pipeline-start-timestamp`                   | Current epoch time              | `CI_PIPELINE_START_TIMESTAMP` | Start time of the pipeline as epoch time. Used for pipeline run duration computation.            |
-| `--show-dagger-logs/--hide-dagger-logs`        | `--hide-dagger-logs`            |                               | Flag to show or hide the dagger logs.                                                            |
-
+| Option                                         | Default value                   | Mapped environment variable   | Description                                                                                 |
+| ---------------------------------------------- | ------------------------------- | ----------------------------- | ------------------------------------------------------------------------------------------- |
```
Comment (evantahler) [inline airbyte-ci/connectors/pipelines/README.md:141]: This table was messed up before.

File: airbyte-integrations/connectors/source-klaus-api/acceptance-test-docker.sh
Comment (evantahler) [inline airbyte-integrations/connectors/source-klaus-api/acceptance-test-docker.sh:1]: This file should be 🪓

File: docs/connector-development/testing-connectors/connector-acceptance-tests-reference.md [hunk 2]
```diff
@@ -64,34 +63,29 @@ RUN pip install ./airbyte/integration_code
...
 
 And test via one of the two following Options
 
-### (Prefered) Option 1: Run against the production acceptance test image
+### Option 1 (Preferred): Run against the Airbyte CI test suite
 
-From the root of your connector run:
-```bash
-./acceptance-test-docker.sh
-```
```
Comment (evantahler) [inline docs/connector-development/testing-connectors/connector-acceptance-tests-reference.md:78]: 🪓

File: docs/connector-development/testing-connectors/connector-acceptance-tests-reference.md [hunk 3]
```diff
@@ -64,34 +63,29 @@ RUN pip install ./airbyte/integration_code
...
+Learn how to use and install [`airbyte-ci` here](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md). The `connectors test` command will run unit, integration, and acceptance tests against your connector.
```
Comment (natikgadzhi) [inline docs/connector-development/testing-connectors/connector-acceptance-tests-reference.md:75]:
```suggestion
Learn how to use and install [`airbyte-ci` here](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md). Once installed, `airbyte-ci connectors test` command will run unit, integration, and acceptance tests against your connector. Pass `--name <your_connector_name>` to test just one connector.
```

**Review by natikgadzhi — APPROVED** [review]
Looks good, I've only made a suggestion on wording about how to run tests (but, nitpicky one). Thank you for fixing the docs quickly!

**Comment by octavia-approvington** [comment]
After a careful ML study, <br/> we think this looks okay.  <br/> imagine code being okay

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**